cluster/saltbase/salt/kube-addons/init.sls

```diff
@@ -0,0 +1,79 @@
+{% if pillar.get('enable_cluster_monitoring', '').lower() == 'true' %}
```

Comment:
I'm going to try the "is defined and pillar['enable_cluster_monitoring']"
approach again, but when I tried this last time, it was always true, even
when enable_cluster_monitoring='false'.

On Tue, Jan 20, 2015 at 10:33 AM, Joe Beda <notifications@github.com> wrote:

> In cluster/saltbase/salt/kube-addons/init.sls
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3601#discussion-diff-23243505>
> :
>
> > @@ -0,0 +1,79 @@
> > +{% if pillar.get('enable_cluster_monitoring', '').lower() == 'true' %}
>
> YAML has an explicit bool type that turns into a python bool via salt.. It
> is a little odd that we are doing a string compare here instead.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/3601/files#r23243505>
> .
>